### DISCUSSION TITLE
Atlantis compared to Gruntwork Pipelines

### DISCUSSION THREAD

**Discussion by iantybo** [gruntwork-io/knowledge-base#468] [Help] [answered]
Do you have an example of the feedback Gruntwork Pipelines posts to Merge Requests specifically on GitLab? Atlantis is typically configured to post the `plan` output on commit, and the `apply` output when `atlantis apply` is written in a comment on the MR. What is the workflow with Gruntwork Pipelines to trigger such actions?

---

<ins datetime="2022-06-15T16:53:47Z">
  <p><a href="https://support.gruntwork.io/hc/requests/108781">Tracked in ticket #108781</a></p>
</ins>

**Comment by yorinasub17** [comment, marked as answer]
This is a duplicate of https://github.com/gruntwork-io/knowledge-base/discussions/75.

If the answer there is not satisfactory, you can follow up on the post there!
